arborist/auth.go

```diff
@@ -678,12 +678,12 @@ func authMappingForUser(db *sqlx.DB, username string) (AuthMapping, *ErrorRespon
 		(
 			SELECT usr_policy.policy_id FROM usr
 			INNER JOIN usr_policy ON usr_policy.usr_id = usr.id
-			WHERE usr.name = $1 AND (usr_policy.expires_at IS NULL OR NOW() < usr_policy.expires_at)
+			WHERE LOWER(usr.name) = $1 AND (usr_policy.expires_at IS NULL OR NOW() < usr_policy.expires_at)
 			UNION
 			SELECT grp_policy.policy_id FROM usr
 			INNER JOIN usr_grp ON usr_grp.usr_id = usr.id
 			INNER JOIN grp_policy ON grp_policy.grp_id = usr_grp.grp_id
-			WHERE usr.name = $1 AND (usr_grp.expires_at IS NULL OR NOW() < usr_grp.expires_at)
+			WHERE LOWER(usr.name) = $1 AND (usr_grp.expires_at IS NULL OR NOW() < usr_grp.expires_at)
```

Comment:
this function is called by `handleAuthMappingPOST` and `handleAuthMappingGET`, you lowered the username in `handleAuthMappingGET` only. If i'm reading this right, either the `username` on L701 should be lowered too, or `username` should be lowered in `handleAuthMappingPOST`

Reply:
Addressed in this commit: https://github.com/uc-cdis/arborist/pull/165/commits/0019600338c3c26a579b66bd8333226975f542f2